README.md

```diff
@@ -1,66 +1,47 @@
-
-
-**Note:** More information on contributing can be found in the @infor-design/enterprise repo [here](https://github.com/infor-design/enterprise/blob/master/docs/CONTRIBUTING.md).
-
-
 # Infor Design System's Enterprise Components for Angular
 
 This repository serves as the central shared location for a crowd-sourced cross-Infor Angular Typescript repo of Xi control wrapped components.
 
-  - avoids many teams building the same things - inefficient and asking Xi Team for different flavors of the same thing
-  - divide the work to get more done
-  - net new teams get huge gains
-  - great way to learn new things about angular and typescript, and code/architecture conventions
-  - includes wrappers and demos (usage) to get new teams and developers up to speed faster
-  - built-in feedback look from dev to H&L and back
+- avoids many teams building the same things - inefficient and asking Xi Team for different flavors of the same thing
+- divide the work to get more done
+- net new teams get huge gains
+- great way to learn new things about angular and typescript, and code/architecture conventions
+- includes wrappers and demos (usage) to get new teams and developers up to speed faster
+- built-in feedback look from dev to H&L and back
 
 This is a working group supported by the core H&L team,  bringing together the various efforts of teams each creating their own angular directives.
 
-The goal is that this will make Angular easier to support for all teams and easier to extend as the library develops and to prevent multiple teams doing the same work. At the moment the project is being shaped during "Knights of the angular table" calls. Contact andrew.lipetzky@infor.com if interested in joining the call its open to anyone how wants to contribute.
-
-The team is using https://slack.com/ for a communication forum.  We have created a [SoHo XI area | http://sohoxi-dev.slack.com/] for discussions related to the components.  There are https://sohoxi-dev.slack.com/messages/general/ and https://sohoxi-dev.slack.com/messages/angular-components/ channels for those discussion topics.  It is becoming more active every day.
+If you're an Infor employee, you can join our MS Teams Group for updates.
 
-## Current Status
+The goal is that this will make Angular easier to support for all teams and easier to extend as the library develops and to prevent multiple teams doing the same work. At the moment the project is being shaped during "Knights of the angular table" calls.
 
-Agreed on project scaffolding (using CLI and Typescript). Agreed on [shared Repo location |http://jira/browse/SOHO-4016] this master epic has all subtasks related to on going work
-http://jira/browse/SOHO-4016. You can also use it to see whats been done.
+Contact andrew.lipetzky@infor.com if interested in joining the call its open to anyone how wants to contribute.
```

Comment:
Is there a better mechanism than having Andrew's email address exposed? Additionally, he is not really in charge of the call anymore either.

Reply:
I'm not sure. Also there is a lot of Infor stuff in here that is probably hardly relevant or inaccessible to the open source community.

Reply:
I did remove his email to save his inbox for now.